<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="m_backgroundMode">
     <item>
      <property name="text">
       <string>Solid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Horizontal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vertical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rectangular</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Radial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Top Left Diagonal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Top Right Diagonal</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>125</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;First color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>m_color1</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KColorButton" name="m_color1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="color">
      <color>
       <red>70</red>
       <green>90</green>
       <blue>130</blue>
      </color>
     </property>
     <property name="defaultColor">
      <color>
       <red>70</red>
       <green>90</green>
       <blue>130</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>187</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>125</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Second color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>m_color1</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KColorButton" name="m_color2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Second Color:</string>
     </property>
     <property name="color">
      <color>
       <red>70</red>
       <green>90</green>
       <blue>130</blue>
      </color>
     </property>
     <property name="defaultColor">
      <color>
       <red>70</red>
       <green>90</green>
       <blue>130</blue>
      </color>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>150</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>&amp;Background mode:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>m_color1</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
